pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -246,7 +259,11 @@ func (s *Service) Accept(msgType string) bool {
 	return msgType == InvitationMsgType ||
 		msgType == RequestMsgType ||
 		msgType == ResponseMsgType ||
-		msgType == AckMsgType
+		msgType == AckMsgType ||
+		msgType == OldInvitationMsgType ||
```

Comment:
I'd prefer doing a string replace of `did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/` => `https://didcomm.org/` at a higher layer @Moopli @llorllale

Reply:
https://github.com/hyperledger/aries-rfcs/pull/579#issuecomment-804982644